yarn.lock

```diff
@@ -25135,6 +25250,18 @@ vite@2.4.4:
   optionalDependencies:
     fsevents "~2.3.2"
 
+vite@2.6.9:
```

Comment:
I didn't realise we could use vite with fozzie components yet!

Reply:
We can use it for JS services – for anything using SFC structure, we're going to look at moving to Vite shortly when we update to Vue 3 (as we need to look at what things we rely on that may need to be handled by Vite)

Reply:
We'll be able to vite for this component, wait for the next PR!